Marlin/Marlin_main.cpp

```diff
@@ -263,6 +263,7 @@ static millis_t max_inactive_time = 0;
 static millis_t stepper_inactive_time = DEFAULT_STEPPER_DEACTIVE_TIME * 1000L;
 millis_t print_job_start_ms = 0; ///< Print job start time
 millis_t print_job_stop_ms = 0;  ///< Print job stop time
+static millis_t last_command_time = 0;
```

Comment:
It's not needed to send the command again because it's already send. Else the firmware should send a resend. The communication can fail because of different length of usb-plugs or other things.

When the firmware sends a wait, it only communicate that it has nothing to do.